docs-v2/content/contributing/release-guide.md

```diff
@@ -0,0 +1,57 @@
+# Release Guide
```

Comment:
This has been updated recently, see #1882

Reply:
Updated